webapi/src/main/scala/org/knora/webapi/responders/v2/search/gravsearch/mainquery/GravsearchMainQueryGenerator.scala

```diff
@@ -217,7 +217,14 @@ object GravsearchMainQueryGenerator {
                 StatementPattern.makeInferred(subj = mainAndDependentResourceVar, pred = IriRef(OntologyConstants.KnoraBase.HasValue.toSmartIri), obj = mainAndDependentResourceValueObject),
```

Comment:
> // WHERE patterns for statements about the main and dependent resources' values

Could you change the comment accordingly (no standoff querying here)?

Reply:
Done in abbabca.